Source/Services/StreamOfBatchedMessagesSender.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Google.Protobuf;
+
+namespace Dolittle.Runtime.Services;
+
+/// <summary>
+/// Represents an implementation of <see cref="ISendStreamOfBatchedMessages{TBatch,TData}"/>.
+/// </summary>
+/// <typeparam name="TBatch">The <see cref="Type"/> of the batch message.</typeparam>
+/// <typeparam name="TData">The <see cref="Type"/> of the data to put in a batch.</typeparam>
+public class StreamOfBatchedMessagesSender<TBatch, TData> : ISendStreamOfBatchedMessages<TBatch, TData>
```

Comment:
Since this thing is really just a class holding a method. Would it be nicer if it was just a static class with a method on it? Not sure if there are like .NET guidelines for these cases - but we don't really need to allocate a class to do this function.

Reply:
Agree. I'm too unsure about the guidelines for this